src/recursive_sorting/recursive_sorting.py

```diff
@@ -1,17 +1,39 @@
-# TO-DO: complete the helpe function below to merge 2 sorted arrays
-def merge( arrA, arrB ):
+# TO-DO: complete the helper function below to merge 2 sorted arrays
+def merge(arrA, arrB):
+    if len(arrA) == 0:
+        return arrB
+    if len(arrB) == 0:
+        return arrA
     elements = len( arrA ) + len( arrB )
     merged_arr = [0] * elements
-    # TO-DO
-    
+
+    idx_a, idx_b = 0, 0
+    for i in range(elements):
+        # if either array is empty, return the other
+        if len(arrA) == idx_a:
+            for j in range(idx_b, len(arrB)):
+                merged_arr[i + j - idx_b] = arrB[j]
+            return merged_arr
+        elif len(arrB) == idx_b:
```

Comment:
it almost seems like you can combine both these if statements into something like...
```
# If one array is at the end, just combine them
if len(arrA) == idx_a or len(arrB) == idx_b:
    if idx_a < len(arrA):
        merged_arr += arrA[idx_a:]
    if idx_b < len(arrB):
        merged_arr += arrB[idx_b:]
    return merged_arr
```